packages/ui-toolkit/src/components/atoms/Pill/Pill.tsx

```diff
@@ -83,7 +83,7 @@ Pill.defaultProps = {
   onClick: () => {},
   isSelected: false,
   isOutlined: true,
-  isAccented: false,
+  isAccented: true,
```

Comment:
rename to isAccent

Reply:
refactor to 
Pill.defaultProps = {
  size: SIZES.BASE,
  onClick: () => {},
  isSelected: false,
  isOutlined: true,
  isAccent: false,
  leadingIcon: null,
  trailingIcon: null
} as DefaultProps;

Reply:
done.